skiko/src/jsMain/kotlin/org/jetbrains/skia/Picture.js.kt

```diff
@@ -0,0 +1,17 @@
+package org.jetbrains.skia
+
+import org.jetbrains.skia.impl.NativePointer
+
+actual typealias AbortCallback = Int
+
+private external fun _registerCallback(cb: () -> Unit, data: Any?): AbortCallback
+
+internal actual fun registerAbortCallback(abort: (() -> Boolean)?): AbortCallback {
+    if (abort == null) { return 0 }
+
+    val data = arrayOf<Boolean?>(null) // We use array instead of structs due to field name mangling
```

Comment:
@JsName annotation might help with field name mangling. But maybe using array for this is fine.

Reply:
Fixed, thanks 🙏